src/main/java/core/basesyntax/strategy/FruitHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+public interface FruitHandler {
+    int INDEX_FRUIT = 1;
+    int INDEX_AMOUNT = 2;
```

Comment:
don't need to create these constants here, move them to FruitTransactionProcessor and pass in handle() method already parsed fruit's name and its amount